workloads/ric/README.md

```diff
@@ -0,0 +1 @@
+# ric
```

Comment:
@naman-gupta-641 Can you add the steps to deploy this package on Nephio?

Reply:
Added